src/app/code/community/Shopgate/Framework/Model/Payment/Braintr/Abstract.php

```diff
@@ -203,10 +204,12 @@ protected function _addBraintreeInfo($payment, $braintreeId, $riskDataId)
      */
     protected function _addInvoice($paymentInfo)
     {
-        if (!$this->getShopgateOrder()->getIsPaid()) {
+	    if (!$this->getShopgateOrder()->getIsPaid()
```

Comment:
tab chars

Reply:
also remove the unnecessary return spacing

Reply:
Fixed